<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QDialog" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="selectPatientLabel">
           <property name="text">
            <string>Select patient:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxPatient"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="selectEvaluatedDataLabel">
           <property name="text">
            <string>Select evaluated data:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxEvaluatedData"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>#Slice</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dice</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Jaccard</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>FP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>FN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TN</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
